pom.xml

```diff
@@ -74,7 +74,7 @@ THE SOFTWARE.
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-cps</artifactId>
-            <version>2.10</version>
+            <version>2.14-SNAPSHOT</version>
```

Comment:
Aaaah, new `structs` version. Adding an explicit one and switching this to 2.13.